gallery/units_demo.py

```diff
@@ -0,0 +1,97 @@
+# coding: utf-8
+"""
+================================
+Quantities, Units, and Constants
+================================
+
+The purpose of this demo is to demonstrate the capabilities of astropy
+`~astropy.units.Unit`, `~astropy.units.Quantity`, and `~astropy.units.Constant`.
+"""
+
+##############################################################################
+# The astropy `~astropy.units.Quantity` object
+# handles defining, converting between, and performing arithmetic
+# with physical quantities, such as meters, seconds, Hz, etc.
+from astropy import units as u
```

Comment:
`import astropy.units as u`? :angel: 

We should standardize on import conventions for the gallery really...

Reply:
Something to add to the contribution guide!